tests/check-prep/main.go

```diff
@@ -136,18 +130,23 @@ func main() {
 			panic(err)
 		}
 		testEntityAccountIdentifier := types.AccountIdentifier{Address: testEntityAddress}
-		if err := txn.Set([]byte("balance/"+types.Hash(&testEntityAccountIdentifier)+"/"+types.Hash(services.OasisCurrency)), storageEncode(&struct {
-			Account *types.AccountIdentifier `json:"account"`
-			Amount  *types.Amount            `json:"amount"`
-			Block   *types.BlockIdentifier   `json:"block"`
-		}{
-			Account: &testEntityAccountIdentifier,
-			Amount: &types.Amount{
-				// https://github.com/oasisprotocol/oasis-core/blob/v20.8.2/go/oasis-node/cmd/genesis/genesis.go#L534
-				Value:    "100000000000",
-				Currency: services.OasisCurrency,
-			},
-		})); err != nil {
+		if err := txn.Set(
+			[]byte("balance/"+types.Hash(&testEntityAccountIdentifier)+"/"+types.Hash(services.OasisCurrency)),
+			storageEncode(
+				&struct {
+					Account *types.AccountIdentifier `json:"account"`
+					Amount  *types.Amount            `json:"amount"`
+					Block   *types.BlockIdentifier   `json:"block"`
+				}{
+					Account: &testEntityAccountIdentifier,
+					Amount: &types.Amount{
+						// https://github.com/oasisprotocol/oasis-core/blob/v20.8.2/go/oasis-node/cmd/genesis/genesis.go#L534
```

Comment:
Why does this refer to some very old tag, maybe we should update the link?

Reply:
I've changed this and exposed the test entity's general balance as the `TestEntityAmount` variable in `tests/common`.

Reply:
it's good practice to use permalinks so that the line numbers don't drift away.